configs/mesh_wisun_S2LP.json

```diff
@@ -1,6 +1,6 @@
 {
     "config": {
-        "enable-led-control-example": false,
+        "enable-led-control-example": true,
```

Comment:
@artokin Did you have some opinion from this flag or is it OK?

Reply:
led control is tested also for Wi-SUN so it can be enabled.